src/content/docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle.mdx

```diff
@@ -265,3 +310,36 @@ The following table describes each segment of the session trace, with a link to
 Session traces help you identify callbacks in your JavaScript code that execute slowly and block the execution of subsequent calls on the browser's main thread. These calls should execute quickly in order to allow the browser to quickly repaint the page in response to user actions.
 
 Session traces highlight any callbacks longer than 33ms. If called in rapid succession (such as inside a `requestAnimationFrame` loop), callbacks longer than 33ms reduce the frame rate below 30 frames per second. This speed seems sluggish to users.
+
+## Plan for paid consumption
+
+Starting June 10, 2024, session trace data will contribute to your overall monthly data consumption. This means it will be charged per your existing order. This change reflects the storage upgrade to the New Relic database (NRDB) which enables [customizing your sampling rate](#configure-sampling-rates), extending data retention, and improved performance.
+
+Run the following query in [query builder](https://one.newrelic.com/data-exploration/query-builder) to view your current monthly consumption of session trace data. If run from a [parent account](/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure/#hierarchy), the consumption of all child accounts will also be included in the results.
+
+```sql
+FROM NrConsumption SELECT rate(sum(consumption), 1 month) as monthly_SessionTrace_GBs WHERE usageMetric = 'UnbilledBrowserEventsBytes' SINCE '2024-04-25'
+```
+
+<img
+  title="Monthly session trace consumption query"
+  alt="A screenshot of monthly session trace consumption"
+  src={browserSessionTraces}
```

Comment:
Do we have this screenshot? This is the same as the first screenshot on the page.